src/foam/dao/EasyDAO.js

```diff
@@ -914,6 +921,13 @@ model from which to test ServiceProvider ID (spid)`,
         }
       }
 
+      if (this.queryCache) {
```

Comment:
spacing should be: if ( this.queryCache ) {

Reply:
Updated